docs/api/web-contents.md

```diff
@@ -221,14 +221,6 @@ Emitted when `webContents` wants to do basic auth.
 
 The usage is the same with [the `login` event of `app`](app.md#event-login).
 
-### Event: 'media-started-playing'
```

Comment:
I think these events should be kept in `WebContents`, and `did-change-theme-color` can also be added here.